src/components/directory-content.tsx

```diff
@@ -529,6 +544,18 @@ const DirectoryContent = () => {
         // TODO openDialog should also be aware of the dialog's type, not only its subtype, because
         // if/when two different dialogs have the same subtype, this function will display the wrong dialog.
         switch (openDialog) {
+            case NetworkModificationType.COMPOSITE.id:
+                return (
+                    <CompositeModificationEditionDialog
+                        open={true}
+                        titleId={'compositeModification'}
+                        // @ts-expect-error TODO: manage null case(s) here
```

Comment:
fix this

Reply:
Done in a very simple and straightforward way. If it ever happens it will cause a regular error during the fetch.

I didn't change the other TODOs on the same subject that are in the same switch.